src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
+  display: grid;
+  grid-template-columns: repeat(4, $width-card);
+  justify-content: center;
+  gap: 48px;
   flex-wrap: wrap;
 
   max-width: 944px;
   margin: 0 auto;
 
+  @media (max-width: 487px) {
+    grid-template-columns: $width-card;
+  }
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, $width-card);
+  }
+
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, $width-card);
+  }
+
```

Comment:
your code doesn't think correctly with width till 767px, you don't have 1 and 2 columns, i guess you shoul set min, not max width

Reply:
thanks, done